devito/function.py

```diff
@@ -669,8 +625,7 @@ def laplace(self):
         derivative wrt. all spatial dimensions.
         """
         derivs = tuple('d%s2' % d.name for d in self.space_dimensions)
-
-        return sum([getattr(self, d) for d in derivs[:self.ndim]])
+        return Add(sum([getattr(self, d) for d in derivs[:self.ndim]]))
```

Comment:
If it's a true Add, perhaps the ``sum`` isn't even necessary anymore?

Reply:
`sum` is from Python not sympy, stilltrying to figure out where to overload it

Reply:
``Add`` should be able to take as input an arbitrary number of symbols/expressions (``*args``) and sum them up internally